toolkit/tools/pkg/osmodifierlib/modifydefaultgrub.go

```diff
@@ -26,7 +27,7 @@ var grubArgs = []string{
 func modifyDefaultGrub() error {
 	var dummyChroot safechroot.ChrootInterface = &safechroot.DummyChroot{}
 	// Get verity, selinux, overlayfs, and root device values from /boot/grub2/grub.cfg
-	values, err := extractValuesFromGrubConfig(dummyChroot)
+	values, rootDevice, err := extractValuesFromGrubConfig(dummyChroot)
```

Comment:
I'm concerned that this approach will prevent Trident from having any control over what root device is set in the kernel command line. I think we should consider having Trident pass the desired root device to EMU instead

Reply:
I see. Created task https://dev.azure.com/mariner-org/ECF/_workitems/edit/9150 for this and I think I will make it a follow up PR, for now I'd like to first confirm the pipeline works with this change